### DISCUSSION TITLE
airupnp-macos-arm64 is loading libcrypto in an unsafe way

### DISCUSSION THREAD

**Discussion by clarknet** [philippe44/AirConnect#511] [General]
I had AirConnect working on my older iMac, but it is now dead. Worked for last year or two - Amazing and thank you - but now trying to get it to run on MacBook M2 Sonoma 14.2.

Here's the error Im getting: 

Last login: Thu Dec 21 14:10:41 on ttys000
/Users/pclark/Desktop/airupnp-macos-arm64 ; exit;                               
pclark@PRC-Macbook16 ~ % /Users/pclark/Desktop/airupnp-macos-arm64 ; exit;
[14:11:37.135267] main:1355 Starting airupnp version: v1.0.11 (Nov 16 2022 @ 00:34:34)
[14:11:37.135851] main:1362 no config file, using defaults
[14:11:37.136465] Start:1087 Binding to 10.0.0.24:49152
WARNING: /Users/pclark/Desktop/airupnp-macos-arm64 is loading libcrypto in an unsafe way
zsh: abort      /Users/pclark/Desktop/airupnp-macos-arm64

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Process completed]

**Comment by philippe44** [comment]
Try the -static option or look at the READM regarding macOS and openSSL

**Comment by clarknet** [comment]
Thanks. Will do. Your my hero of my Sonos System.

Cheers,
Paul Clark

**Comment by clarknet** [comment]
Working great! Thanks. How do I send you $20, Venmo?

**Reply by philippe44** [reply]
Cool - Thank you for the intent, but I don't take donation. Feel free to give to a charity of your choice on my behalf.

**Comment by clarknet** [comment]
Understand and will do!
Thanks again,
Paul
